public_html/templates/index.html

```diff
@@ -71,7 +71,7 @@
 					<div class="row-div col-sm-3 text-center wikiproject-div">
 						{% if row.wikiprojects is not empty %}
 							{% for wp in row.wikiprojects %}
-								<span class="row-div wproject">{{ wp }}</span>
+								<a class="row-div wproject" href="#">{{ wp }}</a>
```

Comment:
When I was working on this app in the very early stages I thought I'd color each Wikiproject bubble uniquely, so every project has an associated color and that's the color you see in the SelectJS too. I slowly realized this was not worth the amount of work behind it though.